src/confluent_kafka/src/Admin.c

```diff
@@ -328,7 +328,94 @@ static int Admin_set_replica_assignment (const char *forApi, void *c_obj,
 
         return 1;
 }
+static int
+Admin_incremental_config_to_c(void *c_obj, PyObject *dict){
+        Py_ssize_t pos = 0;
+        PyObject *ko, *vo;
+        while(PyDict_Next(dict,&pos,&ko,&vo)){
+                PyObject *ks, *ks8;
+                const char *k;
+                const char *v = NULL;
+                int op = -1;
+                rd_kafka_error_t *error;
+                if (!(ks = cfl_PyObject_Unistr(ko))) {
+                        PyErr_Format(PyExc_ValueError,
+                                     "expected config name to be unicode "
+                                     "string");
+                        return 0;
+                }
+                k = cfl_PyUnistr_AsUTF8(ks, &ks8);
+
+                Py_ssize_t iter = 0;
+                PyObject *internal_ko,*internal_vo;
+                PyObject *internal_ks, *internal_ks8;
+                PyObject *internal_vs = NULL, *internal_vs8 = NULL;
+                const char *internal_k;
+                const char *internal_v;
+                while(PyDict_Next(vo,&iter, &internal_ko, &internal_vo)){
+                        if (!(internal_ks = cfl_PyObject_Unistr(internal_ko))) {
+                                PyErr_Format(PyExc_ValueError,
+                                        "expected property name for to be unicode "
+                                        "string");
+                                return 0;
+                        }
+                        internal_k = cfl_PyUnistr_AsUTF8(internal_ks, &internal_ks8);
+
+                        if (!strcmp(internal_k,"operation_type")) {
+                                if (!cfl_PyObject_GetInt(internal_vo, "value", &op, -1, 1)) {
+                                        Py_DECREF(internal_ks);
+                                        Py_XDECREF(internal_ks8);
+                                        return 0;
+                                }
+                        } else if(!strcmp(internal_k,"value")) {
+                                if (!(internal_vs = cfl_PyObject_Unistr(internal_vo)) || !(internal_v = cfl_PyUnistr_AsUTF8(internal_vs, &internal_vs8))) {
+                                        PyErr_Format(PyExc_ValueError,
+                                                "expected value name to be unicode "
+                                                "string");
+                                        Py_XDECREF(internal_vs);
+                                        Py_XDECREF(internal_vs8);
+                                        Py_DECREF(internal_ks);
+                                        Py_XDECREF(internal_ks8);
+                                        return 0;
```

Comment:
1. We should decrease ref count for ks and ks8 as well. 
2. Move these to single `goto err` block.